cmd/auth.go

```diff
@@ -142,12 +142,7 @@ func interactiveLoginInputFunc(_ *cobra.Command, _ []string) {
 
 func askForHostname(t *terminal.Terminal) (string, bool) {
 	previousHost := viper.GetString(config.Hostname)
-	var prefix string
-	if previousHost != "" {
-		prefix = fmt.Sprintf("Hostname (%s): ", previousHost)
-	} else {
-		prefix = fmt.Sprintf("Hostname: ")
-	}
```

Comment:
Even though it's repetitive, it has a bit better readability, which is, in my humble opinion, the more important thing. 

In your approach, you need to think more about what is happening and how it would look like in the output.

Thank you for your attempt, but there is no real need for this change.